security/providers/oidc/src/main/java/io/helidon/security/providers/oidc/TenantAuthenticationHandler.java

```diff
@@ -294,7 +294,7 @@ private AuthenticationResponse errorResponse(ProviderRequest providerRequest,
             String authorizationEndpoint = tenant.authorizationEndpointUri();
             String nonce = UUID.randomUUID().toString();
             String redirectUri =
-                    encode(redirectUri(providerRequest.env()) + "?" + oidcConfig.paramNameTenant() + "=" + tenantId);
+                    encode(redirectUri(providerRequest.env()) + "?" + oidcConfig.tenantParamName() + "=" + tenantId);
```

Comment:
are you sure this whole thing should be encoded? This will encode even the question mark and equals sign.

Reply:
Yes, this is query parameter. This whole value.